webapp/src/main/java/com/box/l10n/mojito/monitoring/PrefixHierarchicalNameMapperStatsdConfigurationProperties.java

```diff
@@ -12,6 +12,7 @@
     String namePrefix = "";
     String tagKeyPrefix = "";
     NamingConvention namingConventionOverride = null;
+    String stripCharacters = "";
```

Comment:
should it be null by default to convey the "nothing to be done", can you have a quick javadoc for format to help with adding the property?

Reply:
Yeah, it makes sense. I'll add the javadoc.